woocommerce/woocommerce
# E2e/update command for e2e

### All Submissions:






### Changes proposed in this Pull Request:

Updates the command to run e2e tests to keep consistent with [the command for running API tests](https://github.com/woocommerce/woocommerce/pull/35283).






### How to test the changes in this Pull Request:



1. Pull branch
2. `pnpm run env:test`
3. `pnpm test:e2e-pw`

Ensure that tests run normally (as if you had run `USE_WP_ENV=1 pnpx playwright test --config=tests/e2e-pw/playwright.config.js`)



### Other information:




### FOR PR REVIEWER ONLY:
